### PR TITLE
feat(platform): Adjusted container command to prevent restarts

### PR DESCRIPTION
**Pull request by amine7777**
https://unique-ch.atlassian.net/browse/UN-8190

### CODE REVIEW

File: charts/sscsid-keeper/Chart.yaml [hunk 1]
```diff
@@ -20,3 +20,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
```
Comment (unique-dominik) [inline charts/sscsid-keeper/Chart.yaml:21]: you must remove ALL previous changes or they will be double

**Review by unique-dominik — CHANGES_REQUESTED** [review]
Changelog messed up, please fix then approve <3

File: charts/sscsid-keeper/Chart.yaml (hunk 1 above)
Comment (amine7777) [reply charts/sscsid-keeper/Chart.yaml:21]: Ahh okii

**Review by unique-dominik — APPROVED** [review]
very nice thanks a ton

**Pull request merged**